### ISSUE TITLE
Shortcut containing "alt" cannot be added

### ISSUE THREAD

**Issue by CelikE** [hluk/CopyQ#2114]
**Describe the bug**
Shortcut containing `alt` cannot be added

**To Reproduce**
1. Go to `Preferences`
2. Click on `Shortcuts`
3. Try adding a shortcut that contains `alt`, such as `alt+Q` (the one that I usually use)
4. `CopyQ` is not registering the shortcut

**Expected behavior**
Expecting that a shortcut would be created when certain the keys are pressed

**Version, OS and Environment**
```
CopyQ Clipboard Manager 6.3.2
Qt: 5.15.6
KNotifications: 5.98.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: EndeavourOS
```

**Comment by hluk** [comment]
Possible fix in #2121 - it uses a **Qt widget to grab shortcuts** instead of a custom one.

Any chance you can tests the fix? You can also install Flatpak image with the fix from: https://github.com/flathub/com.github.hluk.copyq/pull/51 (use the command from the latest flathubbot's comment to install it)

**Comment by CelikE** [comment]
> Possible fix in #2121 - it uses a **Qt widget to grab shortcuts** instead of a custom one.
> 
> Any chance you can tests the fix? You can also install Flatpak image with the fix from: [flathub/com.github.hluk.copyq#51](https://github.com/flathub/com.github.hluk.copyq/pull/51) (use the command from the latest flathubbot's comment to install it)

This worked :+1:
